gcp-pubsub/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("io.micronaut.build.internal.module")
```

Comment:
why do you remove the parenthesis. I think Kotlin DSL will require the parenthesis. and Groovy DSL will work with or without the parenthesis.